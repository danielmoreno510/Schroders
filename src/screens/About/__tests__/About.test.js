import React from 'react';
import TestRenderer from 'react-test-renderer';

import About from '../About';

describe('About component', () => {
  describe('GIVEN a set of valid props', () => {
    describe('WHEN the component is rendered', () => {
      const testRenderer = TestRenderer.create(<About />);

      it('THEN should display a regular About', () => {
        expect(testRenderer.toJSON()).toMatchSnapshot();
      });
    });
  });
});
