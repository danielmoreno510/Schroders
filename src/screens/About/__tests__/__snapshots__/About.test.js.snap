// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component GIVEN a set of valid props WHEN the component is rendered THEN should display a regular About 1`] = `
"<ScrollView style={{...}}>
  <Text style={{...}}>
    Schroders
  </Text>
  <View style={{...}}>
    <Text style={{...}}>
      Technologies
    </Text>
    <View style={{...}}>
      <Text style={{...}}>
        - React Native
      </Text>
      <Text style={{...}}>
        - TypeScript
      </Text>
      <Text style={{...}}>
        - Redux - Saga
      </Text>
      <Text style={{...}}>
        - esLint
      </Text>
      <Text style={{...}}>
        - Test (Jest)
      </Text>
    </View>
  </View>
  <View style={{...}}>
    <Text style={{...}}>
      Tools
    </Text>
    <View style={{...}}>
      <Text style={{...}}>
        - Reactotron
      </Text>
      <Text style={{...}}>
        - Xcode
      </Text>
      <Text style={{...}}>
        - Prettier
      </Text>
    </View>
  </View>
  <Text style={{...}}>
    This project has a coverage of 76%
  </Text>
  <Image source={{...}} style={{...}} />
  <Text style={{...}}>
    In other projects I have implemented continuous integration processes with implementation within appcenter to download the ios and android builds
  </Text>
  <Image source={{...}} style={{...}} />
  <Image source={{...}} style={{...}} />
</ScrollView>"
`;
